backend/api/views/simplePostView.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework import status
+from django.core import serializers
+
+from ..models import post
+from ..serializers import PostSerializer
+
+import json, sys
+#########################################
+# post request, asking for 
+#    title, str
+#    author_id, uuid
+#    description, str
+#    content, str
+#    visibility, str
+#
+#
+##########################################
+@api_view(['POST'])
+def postAuthorPost(request, author_id):
+  request.data['author_id'] = author_id
+  serializer = PostSerializer(data=request.data)
+  if serializer.is_valid():
+    postInstance = serializer.save()
+    # TODO: a function to update url in desire way
+    # not sure what format the url should be like
+    postInstance.url = "testurl"
```

Comment:
we should probably have the url done before we push this through. the URL is of the form 
`http://localhost:8080/author/{AUTHOR_ID}/posts/{POST_ID}`

Reply:
Updated, However the host is `http://127.0.0.1:5454` as requirement